source/manual/zendesk.html.md

```diff
@@ -0,0 +1,84 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Zendesk
+parent: "/manual.html"
+layout: manual_layout
+section: 2nd line
+type: learn
+last_reviewed_on: 2019-08-16
+review_in: 6 months
+---
+
+2nd line Zendesk tickets are technical errors reported by our users - including government publishers -
+and general queries from users of data.gov.uk, for whom 2nd line provides 1st line support.
+
+## Get started
+
+[Create an account](zendesk-create-account), then ask a fellow 2nd liner to assign a new ticket to
+"2nd/3rd Line—Zendesk Administration" asking to give you access to "2nd Line - GOV.UK Alerts and Issues".
+
+When you're logged in, you should be looking at the ['2nd Line--GOV.UK Alerts and Issues' queue][zendesk-queue].
+
+## Priorities
+
+Work on 'High' priority tickets first - these will have been triaged by 2nd line already, so should be
+relevant and actionable - and start with the oldest by date. If external assistance from outside 2nd
+line is required, assign them on the ticket and move on to the next ticket (keeping the ticket in the
+2nd line queue).
+
+Ignore 'Low' priority tickets - these have been triaged by 2nd line for GOV.UK product managers to review.
+
+New tickets are 'Normal' priority by default and require triaging (i.e. a developer reading and categorising
+it).
+
+## Triaging a ticket
+
+If you have any doubts as to the legitimacy or urgency of a ticket, it should be deferred to the delivery manager.
+
+Once you've established that a ticket is legitimate, you should follow [the process in this diagram](zendesk-triage-diagram).
+There is a printout on the 2nd line desk. In general:
+
+- Questions from users of data.gov.uk can be answered straight away (we're their first line of support)
+- General queries are usually handled by 1st line and content support
+- Product requests should be reassigned to "3rd line GOV.UK Product Requests"
```

Comment:
As far as I know, product requests should just be triaged to low where they be prioritised as normal by the PMs. I think it's also worth noting that 2nd line will always respond to the tickets, even if it's just to say that it will be put on the backlog and gets triaged to low immediately afterwards.

Reply:
Thanks, have now added a sentence about always responding to the ticket.

RE the "3rd line GOV.UK Product Requests" advice, this is new advice as hashed out between  myself, Oscar, Ed & Nila - to avoid 2nd line developers having to know exactly which product manager a given feature request should go to. I wasn't aware that putting something in low priority queue does the same thing (although I was aware we've been asked to put in low priority and assign to a specific PM).

Reply:
If the process has changed then perhaps I'm going on out of date information, but I'm basing it on the "Product question" route of the triage diagram that was made a few months ago: https://drive.google.com/file/d/0B72Q_z4wkLglYkVQd01LSWYwNjNha3NrYVVIMF91eXk3NU1r/view

Reply:
Nila is going to see about getting that document changed 👍 probably worth us sending out an update to the GOV.UK devs so they're aware of the new process too.

Reply:
I wonder if it's worth leaving this page to match the old process until that document has changed (and the process change is confirmed), since that's supposed to be the canonical source of the process (as it's printed out on the desks).

I also have some questions about the new process in this case, since it becomes unclear what the point of the low priority part of the 2nd line queue is, but that's not for this PR.

Reply:
Good point. I'll strip this out for now 👍